StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactoryConfig.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  PaymentSheetFormFactoryConfig.swift
+//  StripePaymentSheet
+//
+
+import Foundation
+
+@_spi(STP) import StripePayments
+
+enum PaymentSheetFormFactoryConfig {
+    case paymentSheet(PaymentSheet.Configuration)
+    case customerSheet(CustomerSheet.Configuration)
+
+    var customer: PaymentSheet.CustomerConfiguration? {
+        switch self {
+        case .paymentSheet(let config):
+            return config.customer
+        case .customerSheet:
+            return nil // TODO: Pass in adapter.
+        }
+    }
+    var merchantDisplayName: String {
+        switch self {
+        case .paymentSheet(let config):
+            return config.merchantDisplayName
+        case .customerSheet:
+            return ""
```

Comment:
should this assert too?

Reply:
Sure, it doesn't hurt to add it right now, as a reminder that we'll need to add it when we add support for achv2.